### ISSUE TITLE
An an Authorised Staff I want view sales report base on sorting

### ISSUE THREAD

**Issue by BorderITS** [borderitsolutions/amadaa#87]

**Comment by lcabrini** [comment]
Question: what is an authorized staff and how are they different from a "regular" staff?

**Comment by BorderITS** [comment]
I don’t want use any name since this is going to be a reporting tools for the client

**Comment by BorderITS** [comment]
CEO example

**Comment by lcabrini** [comment]
Yes, but a CEO could be an example of a user role. I'm not sure that there are many reports that should be viewed only be the CEO, but if there is, then CEO is a correct role to place there.

**Comment by BorderITS** [comment]
Ok

**Comment by BorderITS** [comment]
how about this

**Comment by lcabrini** [comment]
OK, so all users should be able to view sales reports? I'm not saying that it cannot be so, I just want to know if that is how we want it to work.

**Comment by BorderITS** [comment]
the sales can view their daily sales report and can also generate previous as well I think that will be better to me
